src/pheval/cli_pheval_utils.py

```diff
@@ -142,42 +135,38 @@ def scramble_phenopackets_command(
     "--analysis",
     "-a",
     required=True,
-    type=click.Choice(["heatmap", "percentage_diff"], case_sensitive=False),
+    type=click.Choice(["heatmap", "percentage_diff", "distribution"], case_sensitive=False),
     help="""There are two types of analysis:
         heatmap - Generates a heatmap plot that shows the differences between the semantic similarity profiles using the
         score column for this purpose. Defaults to "heatmap".
         percentage_diff - Calculates the score column percentage difference between the semantic similarity profiles""",
 )
 @click.option(
-    "--output",
-    "-o",
-    metavar="FILE",
-    default="percentage_diff.semsim.tsv",
-    help="Output path for the difference tsv. Defaults to percentage_diff.semsim.tsv",
+    "--output-folder",
```

Comment:
Maybe this could be changed to `--output-dir` so it follows the same style as the other click command parameters